app/routers/mail.py

```diff
@@ -0,0 +1,38 @@
+from fastapi import BackgroundTasks, APIRouter, Depends
+from pydantic import BaseModel, EmailStr
+from pydantic.errors import EmailError
+
+import config
+from internal.mail import send_fast_email_invitation
+
+router = APIRouter()
+
+CORRECT_EMAIL_ADDRESS_ERROR_MESSAGE = "Please enter valid email address"
+SUCCESSFULLY_SENT_EMAIL_MESSAGE = "Your message was sent successfully"
+
+
+class InvitationParams(BaseModel):
+    sender_name: str
+    recipient_name: str
+    recipient_mail: str
```

Comment:
Why isn't this in the config file?

Reply:
those are the parameters for the body of this route. they are not configurations

as you can see here [https://fastapi.tiangolo.com/tutorial/body/#create-your-data-model](BaseModel)